### ISSUE TITLE
View multiple models with absolute positions (i.e. no centering)

### ISSUE THREAD

**Issue by antoszewski** [3dmol/3Dmol.js#774]
Is there a way to view multiple models (a la [this example](http://3dmol.csb.pitt.edu/tests/auto/generate_test.cgi?test=testLastModel)), but without any implicit centering? 

Specifically, in the linked example, you are viewing PDB 5XVQ and 5XVX, which, when loaded into another rendering program, looks like this: 
![image](https://github.com/3dmol/3Dmol.js/assets/53012990/a14cc663-5ef3-4216-937c-788844e3f4a6)

These PDB coordinates do not overlap, and exist quite a bit away from each other. However, in the above-linked example, they are rendered as being on top of one another. I assume there is some implicit centering here, like taking the geometric center of the molecule and putting it at the center of the viewer. Is there a way to disable this behavior, and instead view the two models with their absolute xyz positions maintained exactly as in the input file? 

Thanks in advance!

**Comment by dkoes** [comment]
Sounds like you want a grid view:
https://3dmol.org/tests/auto/generate_test.cgi?test=testAllenes

https://3dmol.org/doc/global.html#createViewerGrid

**Comment by antoszewski** [comment]
Thanks for the reply, the ViewerGrid does look interesting! For the particular application I had in mind, though, we would have an .sdf file containing two molecules that have been pre-aligned via an external algorithm. We would then want to view those two aligned molecules, superimposed to make it easier to see similarities/differences. It seems like the grid might not be ideal for that application? 

We had been trying to simply add two models like in the example I linked in the original request, but were finding that the alignment was somehow perturbed when we did this. Specifically, the alignment looks great when viewing the file in VMD or PyMol, but in the 3DMol viewer the positions of one (or both) of the molecules had been changed relative to one another, so that the alignment was no longer correct. 

I can attach a test .sdf file showing the aligned ligands, if that would help?

**Comment by dkoes** [comment]
Yes, that would help.  There should be no moving of model coordinates from what is in the input file.

**Comment by antoszewski** [comment]
I think the easiest example is the two files loaded in the link I posted originally: 

http://3dmol.csb.pitt.edu/tests/auto/generate_test.cgi?test=testLastModel

These two files do not share any overlapping coordinates, as shown in the snapshot I attached earlier from Maestro. However, in the view from example, the two models (shown in licorice and cartoon representations) appear to overlay.

**Comment by dkoes** [comment]
But they **do** overlap.  They are the PDBs 5xvq and 5xvk.  Here is how PyMOL renders them.
![image](https://github.com/3dmol/3Dmol.js/assets/2342142/edd63026-a67e-4bfd-b55d-1329f8686483)

**Comment by antoszewski** [comment]
Sorry for the long delay here! I had grabbed the wrong PDBs initially, my fault. An example of the behavior we were seeing, though, was when using the attached SDF file (attached in txt format for upload purposes). This file just contains two exactly identical molecules. 

[test1.txt](https://github.com/3dmol/3Dmol.js/files/14872537/test1.txt)

We used the code snippet below to visualize these two molecules. _data1_ and _data2_ were set to the first and second molecule, with filetype being _sdf_. 


```
viewer.addModel(data1, fileType);
viewer.setStyle( { stick: { scale: 0.25, colorscheme: "Jmol" } });
viewer.addModel(data2,fileType);
viewer.setStyle({model:1}, { stick: { scale: 0.25, 'color':'yellow'}})
viewer.zoomTo();
viewer.render();
```

When doing this, we got a view reproduced below: 
![11](https://github.com/3dmol/3Dmol.js/assets/53012990/a2a05fb1-c5d8-406f-97c2-e7f06e1c38a5)

I have two questions about this view:

1. Why don't these molecules exactly overlap as one might expect? 
2. Why does the second model (in yellow) not show hydrogens, while the first model (in Jmol) does? Even if you change the second model to colorscheme: Jmol, we see the same thing. Is there a way to manually set which hydrogens are viewable (i.e., all, none, or just polar hydrogens)?

**Comment by dkoes** [comment]
I cannot reproduce any of these issues.
https://colab.research.google.com/drive/14dvsYv9rnhewkuLCCuNv7r3Hst-_kf8y?usp=sharing

Since scale isn't a supported option to stick but seems to be working in your example, I strongly suspect you are not using the latest release.

**Comment by antoszewski** [comment]
I see, thank you! We ended up removing the scale option and cleaning up the quotation marks in the other options, and things then seemed to work. Perhaps it was a versioning issue, although we were following the instructions [here](https://3dmol.org/doc/tutorial-code.html). Regardless, cleaning up the options fed into `setStyle` seemed to fix our problems. We then added the `keepH` flag to `addModel` which controlled the hydrogen behavior. Thanks for all of your help!
